.github/workflows/ci.yml

```
@@ -36,7 +36,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [ "3.12", "3.13" ]
+        python-version: [ "3.12", "3.13", "3.14" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)